drivers/gpio/gpio_npcx.c

```diff
@@ -84,6 +85,14 @@ static int gpio_npcx_config(const struct device *dev,
 	if ((flags & GPIO_OUTPUT) == 0)
 		inst->PDIR &= ~mask;
 
+	/*
+	 * If this GPIO pad is configured for low-voltage power supply, force
+	 * it as open-drain type.
```

Comment:
Update the comment to note that the NPCX GPIO module requires the open drain setting for low-voltage pins.

Reply:
Done.

> this should have a motivation here, since Zephyr doesn't use

Got it. I will add more description in the commit message later.

Reply:
Done.